examples/resources/rafay_aks_cluster_v3/testmodule.sh

```diff
@@ -0,0 +1 @@
+rm -rf .terraform* && rm -rf terraform.tfstate && terraform init && terraform init && terraform plan
```

Comment:
Not sure we want to have the shell script in the example folder, This is an external-facing repo. Move this to tools folder

Reply:
removed the file.